Sources/ApolloCodegenLib/Frontend/JavaScript/package.json

```diff
@@ -13,7 +13,7 @@
     "npm": ">=7"
   },
   "dependencies": {
-    "graphql": "^16.0.1",
+    "graphql": "16.3.0-canary.pr.3510.5099f4491dc2a35a3e4a0270a55e2a228c15f13b",
```

Comment:
I would love to merge to master and have CCN be part of a release, and I agree that relying on me to put out updated canary releases is not ideal. The current blocker is that Ivan is the sole active person with maintainer permissions for the GraphQL.js. No one else readily involved in the project has permissions to merge or do releases. There are some folks from The Guild who would be interested in helping out, so I'm going to float that at the next working group meeting.

I'll leave it up to yall if you want to hold off on merging until we get an official release or not.

Reply:
@AnthonyMDev @twof I just talked to Ivan about this; each canary PR release has an npm tag assigned that follows the format of `canary-pr-XXXX`. This means we can use `canary-pr-3510` here instead.

Reply:
I can push breaking changes to the branch at any time. Any idea if that will cause problems if you depend on a release based on a branch rather than a specific commit? I'm not the most familiar with NPM.

Reply:
I imagine it would. I'm not super familiar with NPM either. Can we create a stable branch for CCN and point to that instead?

Then you can continue to push breaking changes to your branch for development, but only merge that dev branch into the stable CCN branch when you are mostly confident it won't break things.

Reply:
Yeah I can give that a shot.